bridge/bridge/nem/NemUtils.py

```diff
@@ -20,15 +20,21 @@ def calculate_transfer_transaction_fee(xem_amount, message=None):
 
 # region extract_wrap_address_from_transaction
 
-def _process_transfer_transaction(is_valid_address, transaction_identifier, transaction_json):
+def _process_transfer_transaction(is_valid_address, mosaic_id, transaction_identifier, transaction_json):
 	amount = 0
 	transaction_version = transaction_json['version'] & 0x00FFFFFF
 	if 1 == transaction_version or 0 == len(transaction_json['mosaics']):
+		if mosaic_id:
+			return make_wrap_error_result(
+				transaction_identifier,
+				f'mosaic {mosaic_id[0]}:{mosaic_id[1]} must be transferred in mosaic bag')
```

Comment:
Should there be a different error for when the transaction version is 1?  I guess similar to the error above 
``raise ValueError('version one does not support custom mosaics')``

But I assume this will only happen for users who create the transfer tx.

Reply:
this is for a bridge address that accepts a custom mosaic. this is error registered because it was sent native XEM.

Reply:
Yes, the error message could be clearer, as I was unsure what a 'mosaic bag' meant without looking at the code. 😄 

 What you said above is better.

``Native XEM is not supported``